src/test/java/com/synopsys/integration/alert/web/api/job/JobConfigActionsTest.java

```diff
@@ -0,0 +1,856 @@
+    private DescriptorMap descriptorMap;
+
+    @BeforeEach
+    public void init() {
+        authorizationManager = Mockito.mock(AuthorizationManager.class);
+        descriptorAccessor = Mockito.mock(DescriptorAccessor.class);
+        configurationAccessor = Mockito.mock(ConfigurationAccessor.class);
+        jobAccessor = Mockito.mock(JobAccessor.class);
+        fieldModelProcessor = Mockito.mock(FieldModelProcessor.class);
+        descriptorProcessor = Mockito.mock(DescriptorProcessor.class);
+        configurationFieldModelConverter = Mockito.mock(ConfigurationFieldModelConverter.class);
+        globalConfigExistsValidator = Mockito.mock(GlobalConfigExistsValidator.class);
+        pkixErrorResponseFactory = Mockito.mock(PKIXErrorResponseFactory.class);
+        descriptorMap = Mockito.mock(DescriptorMap.class);
+
+        Mockito.when(authorizationManager.hasCreatePermission(Mockito.any(), Mockito.any())).thenReturn(true);
+        Mockito.when(authorizationManager.hasReadPermission(Mockito.any(), Mockito.any())).thenReturn(true);
+        Mockito.when(authorizationManager.anyReadPermission(Mockito.any(), Mockito.any())).thenReturn(true);
+        Mockito.when(authorizationManager.hasWritePermission(Mockito.any(), Mockito.any())).thenReturn(true);
+        Mockito.when(authorizationManager.hasDeletePermission(Mockito.any(), Mockito.any())).thenReturn(true);
+        Mockito.when(authorizationManager.hasExecutePermission(Mockito.any(), Mockito.any())).thenReturn(true);
+    }
+
+    @Test
+    public void createTest() throws Exception {
+        UUID jobId = UUID.randomUUID();
+        FieldModel fieldModel = createFieldModel();
+        JobFieldModel jobFieldModel = new JobFieldModel(jobId.toString(), Set.of(fieldModel));
+        ConfigurationFieldModel configurationFieldModel = ConfigurationFieldModel.create(ProviderDescriptor.KEY_PROVIDER_CONFIG_NAME);
+        ConfigurationJobModel configurationJobModel = new ConfigurationJobModel(jobId, Set.of(createConfigurationModel()));
+
+        Mockito.when(fieldModelProcessor.performBeforeSaveAction(Mockito.any())).thenReturn(fieldModel);
+        Mockito.when(configurationFieldModelConverter.convertToConfigurationFieldModelMap(Mockito.any())).thenReturn(Map.of("Key", configurationFieldModel));
+        Mockito.when(jobAccessor.createJob(Mockito.anyCollection(), Mockito.anyCollection())).thenReturn(configurationJobModel);
+        Mockito.when(configurationFieldModelConverter.convertToFieldModel(Mockito.any())).thenReturn(fieldModel);
+        Mockito.when(fieldModelProcessor.performAfterSaveAction(Mockito.any())).thenReturn(fieldModel);
+
+        JobConfigActions jobConfigActions = new JobConfigActions(authorizationManager, descriptorAccessor, configurationAccessor, jobAccessor, fieldModelProcessor, descriptorProcessor, configurationFieldModelConverter,
+            globalConfigExistsValidator,
+            pkixErrorResponseFactory, descriptorMap);
```

Comment:
Looks like some strange formatting here. We should keep them all one line or put them all on separate lines

Reply:
save actions for some reason broke them up, I changed it manually and it seems to remain. I will update it for all of the cases its used in.